packages/ui-mail-template/src/templates/layout-header.pug

```diff
@@ -1,3 +1,13 @@
-tr
-  td(style={ padding: '12px 24px' })
-    img(src='https://e7.pngegg.com/pngimages/605/905/png-clipart-free-pic-web-design-label-thumbnail.png' width='100px')
+tr(
+    style={
+      width: '100%',
+      'background-color': "#380559"
+    }
+)
+    td(style={ padding: '20px 0px', 'text-align': 'center', href: 'xpon.ai' })
+      a(
+            href="https://xpon.ai/" 
```

Comment:
Spacing look not good !
<img width="1139" alt="image" src="https://github.com/xponvn/test-assessment/assets/44595953/4222e461-e071-4a32-93d9-23bd4fddac2b">

Reply:
resolved. Pls check again. Thanks